src/components/SocialButtons/socialAuth.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import GoogleLogin from 'react-google-login';
+import FacebookLogin from 'react-facebook-login';
+import axios from 'axios';
+import config from '../../config.json';
+
+const URL = process.env.REACT_APP_API_URL;
+let provider = '';
+
+export const responseHandler = (response) => {
+  const url = `${URL}users/social_auth/`;
+  const { accessToken } = response;
+  if (response.email) {
+    provider = 'facebook';
+  } else {
+    provider = 'google-oauth2';
+  }
+  axios.post(url, {
```

Comment:
to avoid doing this and also importing axios in every file , you can have a look at how you can create an axios instance with all the configuration then utilise that. you can also have actions triggered here that make the API call . [axios instance](https://github.com/axios/axios#creating-an-instance)